aws/kinesis/core/kinesis_producer.cc

```diff
@@ -137,6 +139,7 @@ make_sdk_client_cfg(const aws::kinesis::core::Configuration& kpl_cfg,
   cfg.executor = sdk_client_executor;
   cfg.verifySSL = kpl_cfg.verify_certificate();
   cfg.caPath = ca_path;
+  cfg.caFile = ca_file;
```

Comment:
Lets only set this when its not empty

Reply:
> Lets only set this when its not empty

In the [AWS C++ SDK](https://github.com/aws/aws-sdk-cpp/blob/main/src/aws-cpp-sdk-core/include/aws/core/client/ClientConfiguration.h#L290), it is defined as 
> ```Aws::String caFile;```

where Aws::String is aliased to std::string. Setting an empty string would take no effect, so there is no need to check if ca_file is empty or not.

Reply:
ack, resolving this thread